content/tutorials/Team2Tutorial.md

```diff
@@ -0,0 +1,259 @@
+---
+title: ESP32 Communicaation Over WiFi !
+date: 2024-12-06
+authors:
+  name: Anton John Del Mar
+  name: Daniel Chen
+  name: Peter Quawas
+---
+
+<img src = "image-5.png" width = "590" height = "300" />
+
+## Introduction
+
+In this tutorial we will teach you how to use a python script to communicate with your ESP32! In a previous tutorial you learn how to send signals over the serial peripheral interface (SPI), now we will teach you how to send signals wirelessly. We hope this short simple tutorial can help kickstart your project so you don't have to tirelessly search online watching YouTube videos and reading forums like we did!
+
+### Learning Objectives
+
+- Coding using Python / Arduino IDE
+  - Learn about sockets and wireless access points
+
+### Background Information
+
+We will discuss how to setup your ESP32 as a SoftAP (software enabled access point) and how you can use sockets in Python to communicate to your ESP32 over this wireless access point. We use the WiFi capability of the ESP32 as one of the main methods for wireless communication. The key concepts you should understand beforehand are basic coding skills using Python/Arduino, but it is helpful to read into SoftAP and sockets before starting.
+
+## Getting Started
+
+You should have the Arduino IDE and your preferred Python IDE already downloaded, but if not I would recommend using Visual Studio Code. This is all you will need for the software side of things and this will suffice for both Windows and Mac. In terms of hardware, all you will need is your ESP32 and motor shield / motor provided in the previous tutorial (Spinning and Blinking).
+
+### Required Downloads and Installations
+
+[Arduino IDE](https://www.arduino.cc/en/software)
+[Visual Studio Code](https://code.visualstudio.com/download)
```

Comment:
In Markdown, source newlines don't result in rendered newlines, so these two links are on the same line. To force a newline between them, insert an empty line between them.

Reply:
I fixed this issue in the most recent update.